### ISSUE TITLE
A11y_AzureDataStudio_Dashboard_Databases_ScreenReader : The expand/collapse state is not announced for "Show Action" control for screen reader user.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11553]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudio_July2020;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev2;#Benchmark;#MAS1.3.1;#MAS4.2.1;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.2.1

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Double click on connected server or right click on it & select manage option to open the Dashboard.
4. Navigate to Databases under Dashboard & hit enter.
5. Start screen reader, navigate to the "Show Action" control and observe if the expand/collapse state is announced or not.

### Actual:
When screen reader users navigate to the "Show Action" control, it's state is announced as 'selected' which is incorrect state that is announced.

This issue is observe in the below scenarios:
1. Notebook - Markdown Toolbar -> The expand/collapse state is not announced for "Cell" control for screen reader user.

### Expected:
The "Show Action" control should be provided with expand/collapse state for screen reader users so that users are able to identify it's state on interacting with it.

### User Impact:
If proper state of the control is not announced to the scree reader users the they will not understand how to interact with that control.

### Attachment link for Reference:
[11553_A11y_AzureDataStudio_Dashboard_Databases_ScreenReader The expand:collapse state is not announced for "Show Action" control for screen reader user.zip](https://github.com/microsoft/azuredatastudio/files/4986999/11553_A11y_AzureDataStudio_Dashboard_Databases_ScreenReader.The.expand.collapse.state.is.not.announced.for.Show.Action.control.for.screen.reader.user.zip)

**Comment by alanrenmsft** [comment]
for this particular issue, the ... button is not meant to be a button with expanded/collapsed state. it is just providing an extra way to open the context menu. closing as won't fix.

**Comment by alanrenmsft** [comment]
dupe of #10650
